src/material/legacy-core/public-api.ts

```diff
@@ -29,18 +29,18 @@ export {
   mixinErrorState,
   HasInitialized,
   mixinInitialized,
-  MAT_DATE_LOCALE,
-  MAT_DATE_LOCALE_FACTORY,
+  MAT_DATE_LOCALE as MAT_LEGACY_LOCALE,
+  MAT_DATE_LOCALE_FACTORY as MAT_LEGACY_LOCALE_FACTORY,
   DateAdapter,
   MatDateFormats as MatLegacyDateFormats,
-  MAT_DATE_FORMATS,
+  MAT_DATE_FORMATS as MAT_LEGACY_DATE_FORMATS,
   NativeDateAdapter,
-  MAT_NATIVE_DATE_FORMATS,
+  MAT_NATIVE_DATE_FORMATS as MAT_LEGACY_NATIVE_DATE_FORMATS,
   NativeDateModule,
   MatNativeDateModule as MatLegacyNativeDateModule,
   ShowOnDirtyErrorStateMatcher,
   ErrorStateMatcher,
-  MatLine,
+  MatLine as MatLegacyLine,
   setLines,
   MatLineModule as MatLegacyLineModule,
   RippleGlobalOptions,
```

Comment:
We should get `MAT_RIPPLE_GLOBAL_OPTIONS` too. I mean actually we should probably rename all of the symbols... if its not too difficult to fit into the copybara and the migration scripts